### DISCUSSION TITLE
Codespace Icons not loading on Firefox

### DISCUSSION THREAD

**Discussion by XxMicrowavexX** [community/community#47177] [Codespaces]
### Select Topic Area

Bug

### Body

When I open a codespace for the first time it works normally. But when I refresh it it gives me a Network Error. I did some research and found that it is caused by CORS so I used the CORS everywhere add on. It loads now but all the icons are missing
![Screen Shot 2023-02-13 at 1 44 11 PM](https://user-images.githubusercontent.com/76092869/218347570-d5acda4c-0576-4fb5-a049-d0be9d6b22b7.png)

**Comment by KartikM123** [comment]
Hi @XxMicrowavexX ! Thanks for reaching out!

Do you mind sending any CORS/network issues in your dev console? Are you still able to repro this? I was able to successfully load the codespace on my firefox browser, so I wonder if it's a browser or network specific issue.

**Reply by XxMicrowavexX** [reply]
.

**Comment by XxMicrowavexX** [comment]
Without CORS everywhere
![image](https://user-images.githubusercontent.com/76092869/218831247-df54435b-4c6e-42dc-bb4a-717a337d0f21.png)
With CORS everywhere
![image](https://user-images.githubusercontent.com/76092869/218831538-498cd751-3343-4097-b956-77caab78337f.png)


I'm guessing that the add-on isn't working properly?

**Comment by XxMicrowavexX** [comment]
Out of boredem I decided to try codespaces again. This time the same icons thing happensbut now I get this error:
An unexpected error occurred that requires a reload of this page.
The workbench failed to connect to the server (Error: )
